sunpy/coordinates/tests/test_wcs_utils.py

```diff
@@ -312,7 +312,8 @@ def test_frame_mapping_obsgeo_spherical(dkist_location, caplog):
     loc_sph = location.spherical
     wcs = WCS(naxis=2)
     wcs.wcs.ctype = ['HPLT', 'HPLN']
-    wcs.wcs.obsgeo = [0, 0, 0] + [loc_sph.lon.to_value(u.deg), loc_sph.lat.to_value(u.deg), loc_sph.distance.to_value(u.m)]
+    wcs.wcs.obsgeo = [0, 0, 0] + \
+        [loc_sph.lon.to_value(u.deg), loc_sph.lat.to_value(u.deg), loc_sph.distance.to_value(u.m)]
```

Comment:
Can we turn off auto line wrapping outside of parenthesis? Line continuation characters are evil.

Reply:
I will work that out.